### PR TITLE
fix: fix casing for list tokens, reactors, proxies responses

### PR DESCRIPTION
**Pull request by lcschy**
## Description

- Adds proper casing support if response is list for Tokens, Reactors, Proxies

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: src/common/utils.ts [hunk 1]
```diff
@@ -102,42 +103,100 @@ const transformTokenRequestSnakeCase = (
...
   } as Token;
 };
 
 const transformReactorResponseCamelCase = (
-  reactor: Reactor
-): Reactor | undefined => {
-  if (typeof reactor === 'undefined') {
+  reactorResponse: Reactor | PaginatedList<Reactor>
+): Reactor | PaginatedList<Reactor> | undefined => {
+  if (typeof reactorResponse === 'undefined') {
```
Comment (kevinperaza) [inline src/common/utils.ts:145]: Same here, should we update the functions type signature to reflect that `reactorResponse` can also be `undefined`?

File: src/common/utils.ts [hunk 2]
```diff
@@ -102,42 +103,100 @@ const transformTokenRequestSnakeCase = (
   } as Token | CreateToken | UpdateToken;
 };
 
-const transformTokenResponseCamelCase = (token: Token): Token | undefined => {
-  if (typeof token === 'undefined') {
+// any as we dont know if its a list or not
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const isList = <T>(arg: any): arg is PaginatedList<T> =>
```
[the remark this answers is not in the document]
Comment (lcschy) [reply src/common/utils.ts]: nooice, done.

File: test/proxies.test.ts [hunk 3]
```diff
@@ -86,5 +87,76 @@ describe('Proxies', () => {
...
       transformedUpdatePayload,
     }));
+
+    test('should list w/o changing config casing', async () => {
+      const randomString = _chance.string();
+      const randomNumber = _chance.integer();
+
+      client.onGet().reply(
+        200,
+        /* eslint-disable camelcase */
```
[the remark this answers is not in the document]
Comment (lcschy) [reply test/proxies.test.ts]: added the values generated by our api and obvious testing values.

**Review by kevinperaza — APPROVED** [review]
🚀

**Pull request merged**